app/controllers/api/v1/groups_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::V1::GroupsController < ApplicationController
+  def index
+    @groups = Group.all
+
+    render json: @groups
+  end
+
+  def show
+    @group = Group.find(params[:id])
```

Comment:
Hi @marceloboth,
I used to do it,
but sometimes people get confused wondering where is @groups from

So nowadays I do this way:
http://craftingruby.com/posts/2015/05/31/dont-use-before-action-to-load-data.html
http://blog.ragnarson.com/2015/09/30/problems-with-typical-rails-controllers-and-before-actions.html

What do you think?

Thanks for your feedback.